### PR TITLE
added ts, tsx, jsx support

### PR DESCRIPTION
**Pull request by everdrone**
- Added support for typescript, typescript react and javascript react.  
- Fixed closing bracket on `test.failing` snippet

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,33 +1,69 @@
...
-        "path": "./snippets/ava.json"
-      },
-      {
-        "language": "javascript",
-        "path": "./snippets/assertions.json"
-      }
-    ]
-  }
-}
+	"name": "ava",
```
Comment (SamVerschueren) [inline package.json:2]: Do not change indentation

File: package.json [hunk 2]
```diff
@@ -1,33 +1,69 @@
...
+				"language": "typescriptreact",
+				"path": "./snippets/ava.json"
+			},
+			{
+				"language": "typescriptreact",
+				"path": "./snippets/assertions.json"
+			}
+		]
+	},
+	"__metadata": {
```
Comment (SamVerschueren) [inline package.json:64]: Remove metadata

File: package.json [hunk 3]
```diff
@@ -1,33 +1,69 @@
...
+		"id": "fd48c037-df6e-41e6-9764-8e9cf9c5ca11",
+		"publisherId": "459b7700-3b3b-4f51-b6b6-a5211342c87d",
+		"publisherDisplayName": "Sam Verschueren"
+	}
+}
```
Comment (SamVerschueren) [inline package.json:69]: Add final newline

**Review by SamVerschueren — CHANGES_REQUESTED** [review]

**Comment by antonvasin** [comment]
Hi. I can jump in and finish this PR if still relevant

**Comment by everdrone** [comment]
Hi all, totally forgot about this, sorry for the inconvenience.
closing.

**Pull request closed without merging**